README.md

```diff
@@ -1 +1,47 @@
-# enumerables
\ No newline at end of file
+![](https://img.shields.io/badge/Morse_decoder-blueviolet)
+
+# Ruby Enumerables
+
+> 
+
+
+## Built With
+
+- Ruby
+
+## Authors
+
```

Comment:
- [ ] I would recommend including the steps to clone and run the project locally to make the README professional.

```
git clone https://github.com/cwaku/enumerables.git
cd enumerables
ruby my_list.rb
```